tests/src/test/java/alluxio/shell/command/SetTtlCommandIntegrationTest.java

```diff
@@ -99,9 +102,43 @@ public void setTtlWithInvalidOperationValue() throws Exception {
   }
 
   @Test
-  public void setTtlNegative() throws Exception {
+  public void setTtlWithDifferentUnitTime() throws Exception {
+    String filePath = "/testFile";
+    FileSystemTestUtils.createByteFile(mFileSystem, filePath, WriteType.MUST_CACHE, 1);
+    Assert.assertEquals(Constants.NO_TTL,
+        mFileSystem.getStatus(new AlluxioURI("/testFile")).getTtl());
+
+    AlluxioURI uri = new AlluxioURI("/testFile");
+    HashMap<String, Long> timeUnits = new HashMap<String, Long>();
```

Comment:
Is there a more "elegant" way for initializing a map in Java? It seems that at some point in every approach we have to use the put method to add key value pairs.